Podfile

```diff
@@ -85,9 +85,9 @@ target 'WooCommerce' do
   pod 'Gridicons', '~> 1.2.0'
 
   # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
-  pod 'WordPressAuthenticator', '~> 9.0'
+#  pod 'WordPressAuthenticator', '~> 9.0'
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/woocommerce/woocommerce-ios/blob/f0fe65c47bcd2ba7ce884e5df6773cff304838cd/Podfile#L88"></span>Layout/CommentIndentation: Incorrect indentation detected (column 0 instead of 2).</td>
    </tr>
  </tbody>
</table>